pubsub/mqtt3/metadata.yaml

```
@@ -0,0 +1,63 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: mqtt3
+version: v1
+status: stable
+title: "MQTT3"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-mqtt3/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "Authenticate using a connection string."
+    metadata:
+      - name: url
+        type: string
+        required: true
+        sensitive: true
+        description: |
+          Address of the MQTT broker.
+          Use the `tcp://`` URI scheme for non-TLS communication. 
+          Use the `ssl://`` URI scheme for TLS communication.
```

Based on the docs it would make sense to tie all the TLS-related/required  fields in this comment.

```suggestion
          Use the `ssl://`` URI scheme for TLS communication. Requires `caCert`, `clientKey`, `clientCert` to be defined.
```